### ISSUE TITLE
Add Status Column to Data Transfer Function

### ISSUE THREAD

**Issue by angeloruggieridj** [dbeaver/dbeaver#20344]
FEATURE: Add a column "Status" that show the progress of the operation when moving data from source to target container.
It could be helpful show also a tick when a table tranfer is completed when you have a lot of transfers in that operation.

DB Technology: Oracle

**Comment by ShadelessFox** [comment]
Hello @angeloruggieridj,

I believe it's already implemented. When you perform data transfer, a progress bar appears at the bottom of the dialog.

Are you referring to something else?

**Comment by angeloruggieridj** [comment]
Yes it is correct but when the transfer is finished for a table, DBeaver just open that transferred table in background.
Imagine the case when you use a single data transfer selecting more than one table, it could be useful add a tick in that list when the transfer of that specific table is completed (like when DBeaver opened in the background that table).

**Comment by ShadelessFox** [comment]
I see, thanks for explaining. There's an entire field for improvements. Our team will discuss it.
